models/Model.php

```diff
@@ -1,32 +1,28 @@
 <?php
-
+// Fichier nécessaire pour avoir les identifiants de la bdd
 include('config/connexion_database.php');
-
-// Retourne une connexion à la base de données
-function getPdo(): PDO
-{
-global $dbhost;
-global $dbname;
-global $dbuser;
-global $dbpswd;
-
-    try{
-        $db = new PDO('mysql:host='.$dbhost.';dbname='.$dbname,$dbuser,$dbpswd,array(PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8', PDO::ATTR_ERRMODE => PDO::ERRMODE_WARNING));
-    }catch(PDOexception $e){
-        die("Une erreur est survenue lors de la connexion à la base de données");
-    }
-
-    return $db;
-}
-
+    
 class Model{
-    protected $db; 
 
-    // Comportement appelé dès qu'on créé une instance de la classe => LE CONSTRUCTEUR
-    // A la naissance de l'objet $db on l'hydrate
     public function __construct()
     {
-        $this->db = getPdo();
+        $this->db = Model::getPdo();
     }
+    
+    // Retourne une connexion à la base de données
+    function getPdo(): PDO
+    {
+        global $dbhost;
+        global $dbname;
+        global $dbuser;
+        global $dbpswd;
 
+            try{
+                $db = new PDO('mysql:host='.$dbhost.';dbname='.$dbname,$dbuser,$dbpswd,array(PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8', PDO::ATTR_ERRMODE => PDO::ERRMODE_WARNING));
+            }catch(PDOexception $e){
+                echo("Une erreur est survenue lors de la connexion à la base de données");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)